### PR TITLE
Feat: add --version command

### PR DESCRIPTION
**Pull request by TomerHeber**
Added --version to the flags.

### CODE REVIEW

File: cmd/terratag/main.go [hunk 1]
```diff
@@ -10,13 +11,21 @@ import (
 	"github.com/hashicorp/logutils"
 )
 
+var version = "dev"
```
Comment (TomerHeber) [inline cmd/terratag/main.go:14]: goreleaser will automatically override this.

Example (when running locally):
```
 terratag2 git:(master) ✗ goreleaser release --snapshot --rm-dist
   • releasing...     
   • loading config file       file=.goreleaser.yml
   • loading environment variables
   • getting and validating git state
      • building...               commit=064ce5b17d4ba23b6510be311385bc7a77014cf1 latest tag=v0.1.45
      • pipe skipped              error=disabled during snapshot mode
   • parsing tag      
   • setting defaults 
   • snapshotting     
      • building snapshot...      version=0.1.45-SNAPSHOT-064ce5b
   • checking distribution directory
   • loading go mod information
   • build prerequisites
   • writing effective config file
      • writing                   config=dist/config.yaml
   • building binaries
      • building                  binary=dist/terratag_darwin_amd64_v1/terratag
      • building                  binary=dist/terratag_windows_amd64_v1/terratag.exe
      • building                  binary=dist/terratag_linux_amd64_v1/terratag
      • building                  binary=dist/terratag_darwin_arm64/terratag
      • building                  binary=dist/terratag_linux_arm64/terratag
   • archives         
      • creating                  archive=dist/terratag_0.1.45-SNAPSHOT-064ce5b_darwin_amd64.tar.gz
      • creating                  archive=dist/terratag_0.1.45-SNAPSHOT-064ce5b_darwin_arm64.tar.gz
      • creating                  archive=dist/terratag_0.1.45-SNAPSHOT-064ce5b_linux_amd64.tar.gz
      • creating                  archive=dist/terratag_0.1.45-SNAPSHOT-064ce5b_windows_amd64.tar.gz
      • creating                  archive=dist/terratag_0.1.45-SNAPSHOT-064ce5b_linux_arm64.tar.gz
   • calculating checksums
   • homebrew tap formula
      • writing                   formula=dist/terratag.rb
   • storing release metadata
      • writing                   file=dist/artifacts.json
      • writing                   file=dist/metadata.json
   • release succeeded after 44.71s
➜  terratag2 git:(master) ✗ dist/terratag_darwin_amd64_v1/terratag
missing tags
Usage: terratag -tags='{ "some_tag": "value" }' [-dir="."]
➜  terratag2 git:(master) ✗ dist/terratag_darwin_amd64_v1/terratag --version
Terratag v0.1.45-SNAPSHOT-064ce5b

```

File: cmd/terratag/main.go [hunk 2]
```diff
@@ -10,13 +11,21 @@ import (
...
 	args, err := cli.InitArgs()
 	if err != nil {
-		log.Println(err)
-		log.Println("Usage: terratag -tags='{ \"some_tag\": \"value\" }' [-dir=\".\"]")
+		fmt.Println(err)
+		fmt.Println("Usage: terratag -tags='{ \"some_tag\": \"value\" }' [-dir=\".\"]")
 		return
 	}
+
+	if args.Version {
```
Comment (TomerHeber) [inline cmd/terratag/main.go:24]: If the --version flag is used print the version and exit.

**Review by HeverFarber — APPROVED** [review]
LGTM. 

I just wondering how he knows to overwrite it

**Comment by TomerHeber** [comment]
> LGTM.
> 
> I just wondering how he knows to overwrite it

@HeverFarber 

goreleaser will add the -ldflags argument while building each binary.
See https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications

**Pull request merged**